.github/ISSUE_TEMPLATE/code-contrib-task.yml

```diff
@@ -41,6 +41,19 @@ body:
           have good logical thinking and the ability to solve complex problems, be proficient in programming
           skills or algorithms
 
+  - type: dropdown
+    id: level
```

Comment:
duplicates field in title？

Reply:
yes, but this is a kind of validation since there is no validation in the PR title.